java/server/src/org/openqa/selenium/grid/distributor/local/Host.java

```diff
@@ -55,7 +55,7 @@
 
 class Host {
 
-  private static final Logger LOG = Logger.getLogger("Selenium Distributor");
+  private static final Logger LOG = Logger.getLogger("Selenium Host");
```

Comment:
I kind of did this deliberately, so everything from the distributor appears in the same set of logs.